remind/cogs/reminders.py

```diff
@@ -72,8 +72,8 @@ def get_default_guild_settings():
 
 
 def _contest_start_time_format(contest, tz):
-    start = contest.start_time.replace(tzinfo=dt.timezone.utc).astimezone(tz)
-    return f'{start.strftime("%a, %d %b %y, %H:%M")}'
+    seconds = int(contest.start_time.replace(tzinfo=dt.timezone.utc).timestamp())
```

Comment:
You need to remove couple of more stuff 
1. `remind.cogs.reminders.Reminders.settz` -> This can be deprecated now as we don't have any use of timezone, `remind/cogs/reminders.py:57` This would also get deprecated along

2. `remind.cogs.reminders.Reminders._make_contest_pages` -> here we are still using timezone, Need to remove the `embed.description` line

Reply:
Please see if this update works or not